libs/martech/martech.js

```diff
@@ -97,49 +98,91 @@ const getDtmLib = (env) => ({
       : env.consumer?.marTechUrl || 'https://assets.adobedtm.com/d4d114c60e50/a0e989131fd5/launch-a27b33fc2dc0-development.min.js',
 });
 
-export default async function init({ persEnabled = false, persManifests }) {
+const setupEntitlementCallback = () => {
+  const setEntitlements = async (destinations) => {
+    const { default: parseEntitlements } = await import('../features/personalization/entitlements.js');
+    return parseEntitlements(destinations);
+  };
+
+  const getEntitlements = () => new Promise((resolve) => {
+    const handleEntitlements = (e) => {
+      window.removeEventListener(ALLOY_SEND_EVENT, handleEntitlements);
```

Comment:
Would setting `{ once: true }` not be an option when adding the event listener, thus eliminating the need to remove it?

Reply:
Good point

Reply:
done